src/ragapp/backend/routers/chat/index.py

```diff
@@ -59,7 +88,7 @@ async def chat(
 @r.post("/request")
 async def chat_request(
     data: ChatData,
-    chat_engine: BaseChatEngine = Depends(get_chat_engine),
+    chat_engine: BaseChatEngine = Depends(create_chat_engine),
```

Comment:
do we actually need this endpoint in RAGapp?

Reply:
removed